sig-cli/charter.md

```diff
@@ -0,0 +1,50 @@
+# SIG CLI Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Command Line Interface SIG (SIG CLI) is responsible for kubectl and
+related tools. This group focuses on command line tools and
+libraries to interface with Kubernetes API's.
+
+### In scope
+
+SIG CLI section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+SIG CLI code include command line tools and binaries for working with Kubernetes
```

Comment:
I suggest "general-purpose command-line tools and binaries" as a way to distinguish these from the special-purpose tools built by other SIGs.